gwardia/templates/gwardia/list_events.html

```diff
@@ -0,0 +1,16 @@
+{% for event in events %}
+    {% if event.is_archival %}
+    <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer bg-orange-300 bg-opacity-25">
```

Comment:
`<li>` is copied 2 times, why?

Reply:
one have changed background (archived have orange background)

Reply:
Hmmm. Maybe add something like that `{% if event.is_archival %}bg-orange-300 bg-opacity-25{% endif %}`...